config/sync/metatag.metatag_defaults.node.yml

```diff
@@ -7,8 +7,9 @@ _core:
 id: node
 label: Content
 tags:
+  keywords: '[node:su_news_topics]'
   title: '[node:title] | [site:name]'
   canonical_url: '[node:url]'
   description: '[node:summary]'
-  og_image: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large]'
-  og_image_url: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large]'
+  og_image: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large:node:su_news_featured_media]'
+  og_image_url: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large:node:su_news_featured_media]'
```

Comment:
Ah whoops. These are on the default node setting. We need a separate setting for news items.